srcJava/com/rti/perftest/IMessagingWriter.java

```diff
@@ -21,7 +20,7 @@
     public boolean waitForPingResponse(long timeout, TimeUnit unit);
     public boolean notifyPingResponse();
     public long getPulledSampleCount();
-    public void wait_for_acknowledgments(Duration_t timeout);
+    public void wait_for_acknowledgments(int sec, int nanosec);
```

Comment:
What about follow a standard between the languages and use the same name: 
nsec or nanosec

Reply:
Done